tests/E2E/docker-compose.yml

```diff
@@ -0,0 +1,58 @@
+version: '3'
+
+services:
+  tests:
+    build: .docker
+    environment:
+      SELENIUM_HOST: selenium-chrome
+      URL: prestashop-web
+    depends_on:
+      - "prestashop"
+      - "selenium-chrome"
+    command: ["/tmp/wait-for-it.sh", "--timeout=360", "--strict", "prestashop-web:80", "--", "/tmp/prepare-prestashop.sh"]
+    tty: true
+    volumes:
+      - .docker:/tmp/docker
+      - ../../:/tmp/data-ps/prestashop/
+    networks:
+      - default
+
+  prestashop:
+    build: .docker/prestashop
+    environment:
+      DB_SERVER: db
+      PS_INSTALL_AUTO: 1
+      PS_COUNTRY: fr
+      PS_DEV_MODE: 0
+      PS_ERASE_DB: 1
+      PS_DOMAIN: prestashop-web
+      PS_FOLDER_ADMIN: admin-dev
+      PS_FOLDER_INSTALL: install-dev
+    depends_on:
+      - "db"
+    command: ["/tmp/wait-for-it.sh", "--timeout=60", "--strict", "db:3306", "--", "/tmp/docker_run.sh"]
+    volumes:
+      - .docker:/tmp/docker
+      - ../../:/tmp/data-ps/prestashop/
+    networks:
+      default:
+        aliases:
+          - prestashop-web
+
+  selenium-chrome:
+    image: selenium/standalone-chrome
+    depends_on:
+      - "prestashop"
+    volumes:
+      - /dev/shm:/dev/shm
+    networks:
+      default:
+        aliases:
+          - selenium-chrome
+
+  db:
+    image: mysql:5.7.22
```

Comment:
Are we sure this tag will always be available? Isn't `mysql:5.7` safer?

Reply:
This tag contains a fix because of a bug with latest version of Mysql container. It's weird bug recommended :/

Reply:
Let's keeping it until the day we cannot use it anymore then. :)